backend/src/main/java/gov/cdc/usds/simplereport/service/DiseaseService.java

```diff
@@ -22,14 +26,24 @@ public class DiseaseService {
   private SupportedDisease fluA;
   private SupportedDisease fluB;
 
+  private final Map<UUID, SupportedDisease> supportedDiseaseMap = new HashMap<>();
+
   public void initDiseases() {
     covid = _supportedDiseaseRepo.findByName("COVID-19").orElse(null);
     fluA = _supportedDiseaseRepo.findByName("Flu A").orElse(null);
     fluB = _supportedDiseaseRepo.findByName("Flu B").orElse(null);
+
+    Optional.ofNullable(covid).ifPresent(sd -> supportedDiseaseMap.put(sd.getInternalId(), sd));
+    Optional.ofNullable(fluA).ifPresent(sd -> supportedDiseaseMap.put(sd.getInternalId(), sd));
+    Optional.ofNullable(fluB).ifPresent(sd -> supportedDiseaseMap.put(sd.getInternalId(), sd));
   }
 
   public List<SupportedDisease> fetchSupportedDiseases() {
-    return (List<SupportedDisease>) _supportedDiseaseRepo.findAll();
+    return _supportedDiseaseRepo.findAll();
+  }
+
+  public Map<UUID, SupportedDisease> getCachedSupportedDiseasesMap() {
```

Comment:
nit: I would call this `getKnownSupportedDiseasesMap()` or similar - `cache` implies we're using some sort of real database cache, instead of this Map shortcut. Mostly worry about it for the day when we add a new disease and someone expects this to return all the diseases in the repo.

Reply:
Done!